pytest_operator/plugin.py

```diff
@@ -0,0 +1,108 @@
+import os
+import pytest
+import asyncio
+from pathlib import Path
+from juju.model import Model as JujuModel
+from aioify import aioify
+from subprocess import run
+
+
+def pytest_addoption(parser):
+
+    parser.addoption("--controller", action="store", help="Juju controller to use")
+
+    parser.addoption("--model", action="store", help="Juju model to use")
+
+    parser.addoption(
+        "--cloud", action="store", help="Juju cloud to use"
+    )
+
+
+class ToolsObj:
```

Comment:
This should maybe renamed to `OperatorTools` to match the fixture.

Reply:
Is this meant to be a class that holds the portions we would eventually share with Zaza? If that's the case should it even be this file at all?